packages/mesos-modules/buildinfo.json

```diff
@@ -6,7 +6,7 @@
   "single_source": {
     "kind": "git", 
     "git": "https://github.com/dcos/dcos-mesos-modules.git", 
-    "ref": "29561f11a12372752b77839be73418d1d75abe63", 
-    "ref_origin": "1.12"
+    "ref": "78b54e46bf2024e8438c7394c78d9b38fe728395", 
+    "ref_origin": "master"
```

Comment:
We just point it to `1.12` recently. Shouldn't we backport the change to the `1.12` branch and keep the `ref_origin` the same instead?

Reply:
This is confusing as `dcos` master branch is pointing to a release branch. It would have made more sense if `dcos` 1.12 branch points to mesos-module 1.12 branch. I guess mesos-module was quick in creating a branch :) I will backport the change to 1.12 branch as well